pyiron_base/project/generic.py

```diff
@@ -1461,12 +1461,12 @@ def _update_jobs_in_old_database_format(self, job_name):
                 for entry in db_entry_in_old_format:
                     self.db.item_update({"project": self.project_path}, entry["id"])
 
-    def export_to_archive(self,destination_path,csv_file_name='export.csv', compress=True):
+    def export_to_archive(self, destination_path, csv_file_name='export.csv', compress=True):
         export_archive.copy_files_to_archive(self.project_path, destination_path,compressed=compress)
         df = export_archive.export_database(self,self.project_path,destination_path)
```

Comment:
```suggestion
        export_archive.copy_files_to_archive(self.path, destination_path, compressed=compress)
        df = export_archive.export_database(self, self.path, destination_path)
```

Reply:
An empty line were added, in the new commit, before the function.

Reply:
This is not able an empty line, it is about `Project().path` vs `Project().project_path`

Reply:
Done!